var/spack/repos/builtin/packages/oce/package.py

```diff
@@ -44,6 +44,8 @@ class Oce(Package):
 
     variant('tbb', default=True,
             description='Build with Intel Threading Building Blocks')
+    variant('X11', default=False,
+            description='Build with X11 enabled')
```

Comment:
You are right, ldd -r libTKCAF.so shows dependency on OpenGL (/usr/lib/x86_64-linux-gnu/mesa/libGL.so.1), this in when X11 dependency is enabled. You'd have to do something like sudo apt-get install freeglut3-dev and build-essential  on Ubuntu.